pkg/plugins/manager/client/client.go

```diff
@@ -188,6 +200,10 @@ func (s *Service) CheckHealth(ctx context.Context, req *backend.CheckHealthReque
 			return nil, err
 		}
 
+		if errors.Is(err, context.Canceled) {
+			return nil, plugins.ErrPluginRequestCanceledErrorBase.Errorf("client: check health request cancelled: %w", err)
```

Comment:
Nit for consistency with variable name:
```suggestion
			return nil, plugins.ErrPluginRequestCanceledErrorBase.Errorf("client: check health request canceled: %w", err)
```